hardhat.config.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env node */
+require('dotenv').config()
+
+const mainnetFork = {
+  url: `https://mainnet.infura.io/v3/${process.env.REACT_APP_INFURA_KEY}`,
+  blockNumber: 17023328,
```

Comment:
Curious why this block number in particular? Or will this get re-configured?

Reply:
It's arbitrary. It was close to the latest when I initially wrote this, that's all.

Reply:
could you add a comment here? or should this query the latest block at runtime and use it?